includes/abstracts/abstract-wc-product.php

```diff
@@ -389,6 +390,17 @@ public function get_backorders( $context = 'view' ) {
 		return $this->get_prop( 'backorders', $context );
 	}
 
+	/**
+	 * Get low stock amount.
+	 *
+	 * @param  string $context What the value is for. Valid values are view and edit.
+	 * @since 3.4.0
```

Comment:
Make sure the since tags on new methods are `3.5.0` - thats when this will ship if ready :)